src/db/Place/place.ts

```diff
@@ -138,81 +156,66 @@ export async function addPlace(placeToAdd: Place): Promise<Place> {
  * */
 export async function updatePlace(id: GooglePlaceID["place_id"]): Promise<Place> {
 	const { _col: placeCol, _connection: placeConn } = await getCollection<Place>("place");
-
 	const { _col: ratingCol, _connection: ratingConn } = await getCollection<Rating>("rating");
 
 	const pipeline = [
-		{
-			$match: {
-				placeID: id,
-				braille: {
-					$exists: 1,
-				},
-				fontReadability: {
-					$exists: 1,
-				},
-				staffHelpfulness: {
-					$exists: 1,
-				},
-				guideDogFriendly: {
-					$exists: 1,
-				},
-				navigability: {
-					$exists: 1,
-				},
-			},
-		},
 		{
 			$group: {
 				_id: id,
-				brailleAvg: {
-					$avg: "$braille",
+				guideDogAvg: {
+					$avg: "$guideDogFriendly",
 				},
-				navigAvg: {
-					$avg: "$navigability",
+				isMenuAccessibleAvg: {
+					$avg: "$isMenuAccessible",
 				},
-				fontAvg: {
-					$avg: "$fontReadability",
+				noiseLevelAvg: {
+					$avg: "$noiseLevel",
 				},
-				staffAvg: {
-					$avg: "$staffHelpfulness",
+				lightingAvg: {
+					$avg: "$lighting",
 				},
-				guideDogAvg: {
-					$avg: "$guideDogFriendly",
+				isStaffHelpfulAvg: {
+					$avg: "$isStaffHelpful",
 				},
+				isBathroomOnEntranceFloorAvg: {
+					$avg: "$isBathroomOnEntranceFloor",
+				},
+				isContactlessPaymentOfferedAvg: { $avg: "$isContactlessPaymentOffered" },
+				isStairsRequiredAvg: { $avg: "$isStairsRequired" },
+				spacingAvg: { $avg: "$spacing" },
 			},
 		},
 	];
 
-	const aggCursor = ratingCol.aggregate<
-		Rating & {
-			brailleAvg: number;
-			navigAvg: number;
-			fontAvg: number;
-			staffAvg: number;
-			guideDogAvg: number;
-		}
-	>(pipeline);
+	const aggCursor = ratingCol.aggregate<Omit<Place, "_id" | "promotion">>(pipeline);
```

Comment:
What does `Omit` do?

Reply:
https://docs.mongodb.com/manual/aggregation/#aggregation-pipeline-example
`Omit` is a utility type that returns a type that contains all of the fields of the type argument except for the fields specified. Here, this will return a type with all of the fields of the `Place` type except for the fields `_id` and `promotion`.

Reply:
oooooo this is pretty cool, thanks for explaining!